### ISSUE TITLE
Form inputs need labels

### ISSUE THREAD

**Issue by oliverjam** [fac25/week2-project-Lisa-Karwan#13]
https://github.com/fac25/week2-project-Lisa-Karwan/blob/3da590348fb6e3676820ee5549eb8c1dbf26d4ab/index.html#L72

Unfortunately the `placeholder` attribute cannot replace proper labels for form inputs. The `placeholder` is generally hard to read, it disappears when you start typing, and it is often not exposed to screen reader users.

It's important you always have a label associated with an input. This should usually be an actual `<label>` element, but in some cases could be an `aria-label` if you are _sure_ that you don't need a visible label.

A bit more info: https://web.dev/label/
